src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -42,6 +43,143 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return _hasPowerlineCharacters.value_or(false);
     }
 
+    Windows::Foundation::Collections::IMap<winrt::hstring, winrt::hstring> Font::FontAxesTagsAndNames()
```

Comment:
It seems like this could also return a `std::map`.

Reply:
Is there a way to declare `std::map` in the idl file? I would like to have that in the idl file to keep it consistent with the way we access other things in the `Font` struct like `HasPowerlineCharacters`